README.md

```diff
@@ -105,19 +106,21 @@ export ECR_CROMWELL_TAG=<some-value>
 export ECR_NEXTFLOW_ACCOUNT_ID=<some-value>
 export ECR_NEXTFLOW_REGION=<some-value>
 export ECR_NEXTFLOW_TAG=<some-value>
-export ECR_WES_ACCOUNT_ID=<some-value>
-export ECR_WES_REGION=<some-value>
-export ECR_WES_TAG=<some-value>
 ```
 
-These environment variables point to the ECR account, region, and tags of the Cromwell engine and WES adaptor containers respectively
+These environment variables point to the ECR account, region, and tags of the Cromwell and Nextflow engine respectively
 that will be deployed for your contexts. They are written as Systems Manager Parameter Store variables when you activate
 your Amazon Genomics CLI account region (`agc account activate`). The `./scripts/run-dev.sh` contains logic to determine the current
 dev versions of the images which you would typically use. You may also use production images, the current values of which will
 be written when you activate an account with the production version of Amazon Genomics CLI. If you have customized containers that you 
 want to develop against you can specify these however you will need to make these available if you wish to make pull requests
 with code that depends on them.
 
+#### Option 2. Running with local release
+To run a release version of Amazon Genomics CLI from your local build, first build your changes and then run `make release`.
+This will create a release bundle `dist/` at this package root directory. Run the `install.sh` script to install your 
```

Comment:
I think we have 2-3 different install.sh scripts. We should specify which one

Reply:
Done! thanks!